sys/net/application_layer/asymcute/asymcute.c

```diff
@@ -107,7 +110,7 @@ static uint16_t _msg_id_next(asymcute_con_t *con)
 static uint8_t _req_type(asymcute_req_t *req)
 {
     size_t len;
-    size_t pos = _len_get(req->data, &len);
+    size_t pos = _len_get(req->data, req->data_len, &len);
     return req->data[pos];
```

Comment:
Might make sense to add an assert here that pos is != 1, pos should have been verified in _on_pkt anyhow but just to be sure...

Reply:
Could you also add a comment to the code why the assertion must be true, please?

Reply:
Done.

Reply:
Thanks!